examples/overplotFiberTrace.ipynb

```diff
@@ -52,8 +52,9 @@
     "from pfs.drp.stella.utils import makeFiberTraceSet\n",
     "\n",
     "dateObs = butler.queryMetadata(\"raw\", [\"dateObs\"], dataId)[0]\n",
-    "\n",
-    "fts = makeFiberTraceSet(butler.get(\"fiberTrace\", dataId, dateObs=dateObs))"
+    "pfsFiberTrace = butler.get(\"fiberTrace\", dataId, ccd=5, filter='r', calibDate='2015-12-22', calibVersion='fiberTrace')#, dateObs=dateObs)\n",
+    "fts = makeFiberTraceSet(pfsFiberTrace)\n",
+    "print fts.size(),' FiberTraces found'"
```

Comment:
Users should not need to know that you do this in two pieces.  As mentioned in the code, this needs to be cleaned up so it looks seamless.